packages/wallets/src/components/AppCard/AppCard.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import classNames from 'classnames';
+import type { THooks } from '../../types';
+import { AppCardBadge } from '../AppCardBadge';
+import { WalletText } from '../Base';
+import type { TGenericSizes } from '../Base/types';
+import { WalletGradientBackground } from '../WalletGradientBackground';
+import { WalletsAppLinkedWithWalletIcon } from '../WalletsAppLinkedWithWalletIcon';
+import './AppCard.scss';
+
+type TProps = {
+    activeWalletCurrency: THooks.ActiveWalletAccount['currency'];
+    appIcon: React.ComponentProps<typeof WalletsAppLinkedWithWalletIcon>['appIcon'];
+    appName?: string;
+    balance?: string;
+    cardSize: Extract<TGenericSizes, 'lg' | 'md' | 'sm'>;
+    device: 'desktop' | 'mobile';
+    isDemoWallet?: boolean;
+    walletIcon: string;
+    walletName?: string;
+};
+
+const AppCard: React.FC<TProps> = ({
+    activeWalletCurrency,
+    appIcon,
+    appName,
+    balance,
+    cardSize = 'md',
+    device = 'desktop',
+    isDemoWallet = false,
+    walletIcon,
+    walletName,
+}) => {
+    return (
+        <div className={classNames('wallets-app-card', `wallets-app-card--border-radius--${cardSize}`)}>
```

Comment:
@heorhi-deriv you don't need classNames here, just use javascript string
```suggestion
        <div className={`wallets-app-card wallets-app-card--border-radius--${cardSize}`}>
```